api.yaml

```diff
@@ -322,13 +322,13 @@ paths:
               #! end !#
               "bodyPassthrough": true
             }
-  /v1/cloud/arnid/{arnid}:
+  /v1/cloud/arnid/{resourceid}:
```

Comment:
Question: Why not use `deprecated` field in swagger for now to communicate with consumers?

Reply:
Additional note:

I am fine with this change, however from consumers perspective this change is confusing for following reasons
* API path is `/v1/cloud/arnid/` looking at this information as consumer i expect `arnid` lookup
* If we are communicating with consumers to pass `resourceid` then i would prefer to communicate new endpoint `/v1/cloud/resourceid/{resourceid}` and mark existing endpoint `/v1/cloud/arnid/{arnid}` as deprecated

Reply:
It's resourceid because we wanted to use the same function behind the scenes, I needed to rename the parameter to match the format and parameter of the new function.

I should clarify that no new parameter is being named here, consumer behavior is unchanged. It's just the pathname. Consumers won't have to change anything here.

Reply:
Also, we are fast following this ticket with the deprecation so this is a temporary state that should not exist for more than a few days.

Reply:
ok, lets leave as is.